database/mongodb/mongodb.go

```diff
@@ -184,10 +207,70 @@ func (m *Mongo) Drop() error {
 	return m.db.Drop(context.TODO())
 }
 
+func (m *Mongo) ensureLockTable() error {
```

Comment:
Let's not use the word table in the mongo driver unless we have to because of an interface :)